nim/basolato/server.nimble

```diff
@@ -0,0 +1,23 @@
+# Package
+
+version       = "0.1.0"
+author        = "Anonymous"
+description   = "A new awesome baspolato package"
+license       = "MIT"
+srcDir        = "."
+bin           = @["server"]
+
+backend       = "c"
+
+# Dependencies
+
+requires "nim >= 1.2.4"
+requires "https://github.com/itsumura-h/nim-basolato >= 0.5.1"
```

Comment:
Is there not a **published** version of `basolato` ?

Reply:
Basolato is still develop version and not published to nimble package yet.